lib/networkCall.js

```diff
@@ -0,0 +1,20 @@
+const axios = require('axios')
+const logger = require('../utils/logger')
+
+const networkCall = async (url, queryParams, method = 'get', body = null, headers = null) => {
+  try {
+    const response = await axios({
+      method: method,
+      url: url,
+      params: queryParams,
+      data: body,
+      headers: headers
```

Comment:
You can use ShortHand syntax here, 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer

Reply:
Thanks for pointing out the shorthand syntax. Actually axios uses `data` keyword for `body` and I suppose `body` word is more familar to developers than `data`. I personally never came across `data` for `body`. So I was thinking to use something like `data: body`
What do you say?

Reply:
body is an argument to the function, where ever we are going to call the function, it will require 4th argument to be of type `object`, the argument name doesn't matter for the user of the function.

If you think body will be a better idea then go with it, else `data` with shorthand syntax will always be a better option.

Reply:
I agree. Will stick to the shorthand syntax.

Reply:
Same for `params` object! @swarajpure

Reply:
please check again @ankurnarkhede